components/src/dynamo/common/tests/multimodal/test_lazy_imports.py

```
@@ -0,0 +1,44 @@
+
+# A fresh interpreter is the only honest way to ask this: once any earlier test
+# has imported torch, an eager import here would go unnoticed.
+_PROBE = """
+import sys
+from dynamo.common.multimodal import {names}
+print("torch" in sys.modules)
+"""
+
+
+def _torch_imported_by(names: str) -> bool:
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)